File: java/src/it/grabz/grabzit/GrabzItFile.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */

package it.grabz.grabzit;

import java.io.FileOutputStream;
import java.io.IOException;

/**
 *
 * @author Administrator
 */
public class GrabzItFile {
    private byte[] bytes;

    public GrabzItFile(byte[] bytes)
    {
        this.bytes = bytes;
    }

    /**
     * Save GrabzItFile to physical file.
     * @param path The path to save the file to
     * @throws IOException
     * @throws Exception
     */
    public void Save(String path) throws IOException, Exception
    {
        FileOutputStream fileOuputStream = null;
        try
        {
            fileOuputStream = new FileOutputStream(path);
            fileOuputStream.write(bytes);
            fileOuputStream.close();
        }
        catch (Exception ex)
        {
            if (fileOuputStream != null)
            {
                fileOuputStream.close();
            }
            throw ex;
        }
    }
}

File: 2.1.5/aspnet/GrabzIt/Screenshots/GrabzItFile.cs
using System.IO;

namespace GrabzIt.Screenshots
{
    public class GrabzItFile
    {
        public GrabzItFile(byte[] bytes)
        {
            Bytes = bytes;
        }

        public byte[] Bytes
        {
            get;
            private set;
        }

        public void Save(string path)
        {
            File.WriteAllBytes(path, Bytes);
        }
    }
}
